Omitted Google+ from 'Settings'->'Share on Social Media'.  Fixes #2032

rajshah9914 opened the pull request:
Fixes #2032

Changes: Removed Google+ share option from Settings->Share on Social Media.
Reason being no Google+ share option on Share tab from header scroll down list. Erasing Google+ is a better solution since it is going to end its feature due 2nd April 2019.
Commented the undesired part.

Screenshots for the change: 
After changes:-

![without_google](https://user-images.githubusercontent.com/34474021/52417352-b6091e00-2b11-11e9-9ee8-b8c5f6c9eae7.png)

Before Modification:-

![share_tab](https://user-images.githubusercontent.com/34474021/52417406-d46f1980-2b11-11e9-8fb1-0c75fe32078b.png)
![share_on_social_media](https://user-images.githubusercontent.com/34474021/52417411-d46f1980-2b11-11e9-9e3c-0d1f4e959572.png)

Objective:- Equalizing Share tab in scroll down list and 'Share on Social Media' tab in settings from scroll down list.

AakashMallik commented on src/components/ChatApp/Settings/ShareOnSocialMedia.js:
```diff
@@ -95,7 +95,7 @@ const ShareOnSocialMedia = props => {
             />
           </div>
-          <div style={props.headingStyle}>
+          {/* <div style={props.headingStyle}>
```
Why comment it when you can remove it? :wink:

rajshah9914 replied:
@AakashMallik I deleted the part as suggested by you, it would be great if you review it once again..Thanks..Actually I was trying to see if the code ran on local host perfectly by commenting and then filed a PR for the same :))  
Now made the requested changes.

AakashMallik commented:
LGTM... Just squash the commits and I guess we are good to go. Good start. :+1:  @rajshah9914

AakashMallik commented:
Wait a sec, it seems like the changes in this file are overlapping with another PR of yours. I guess you messed up a bit while branching out.

rajshah9914 commented:
What should I do now?? @AakashMallik , it would be great if you help me out on this..I created a separate branch other than the one I had sent yesterday .. and then made a PR..

AakashMallik commented:
> What should I do now?? @AakashMallik , it would be great if you help me out on this..I created a separate branch other than the one I had sent yesterday .. and then made a PR..

Tidy up your master branch... 
Undo the changes in **contact.react.js** ( you will have to use git reset command so do a bit of google search on how to reset a specific file to a specific commit. ) I hope it is making sense to you.

rajshah9914 commented:
Ok, will try and send another tidy PR soon..Thanks again for guiding..

AakashMallik commented:
> Ok, will try and send another tidy PR soon..Thanks again for guiding..

No problem buddy. Good luck

rajshah9914 commented:
@AakashMallik Please can you review #2032 once again and check if all is fine, I have changed my code that is not merged #2029 and added the same commit message about the change in my file in contact.react.js . Thus I guess it is going fine now, right??

AakashMallik commented:
@rajshah9914 Squash commits, rest looks good to me...

rajshah9914 commented:
@akshatnitd Please can you review it once again, updated the branch and changed the commit and title..
I also upload my changed screenshot beneath.
![without_google](https://user-images.githubusercontent.com/34474021/52622853-d7cb2200-2ed0-11e9-8ee6-d33f69468f18.png)

rajshah9914 commented:
@akshatnitd , hope this satisfies the needs.. Please can you review again..
